runtime/Jakt/CharacterTypes.h

```diff
@@ -172,30 +172,30 @@ constexpr u32 to_ascii_base36_digit(u32 digit)
 
 }
 
-using AK::is_ascii;
-using AK::is_ascii_alpha;
-using AK::is_ascii_alphanumeric;
-using AK::is_ascii_binary_digit;
-using AK::is_ascii_blank;
-using AK::is_ascii_c0_control;
-using AK::is_ascii_control;
-using AK::is_ascii_digit;
-using AK::is_ascii_graphical;
-using AK::is_ascii_hex_digit;
-using AK::is_ascii_lower_alpha;
-using AK::is_ascii_octal_digit;
-using AK::is_ascii_printable;
-using AK::is_ascii_punctuation;
-using AK::is_ascii_space;
-using AK::is_ascii_upper_alpha;
-using AK::is_unicode;
-using AK::is_unicode_control;
-using AK::is_unicode_noncharacter;
-using AK::is_unicode_scalar_value;
-using AK::is_unicode_surrogate;
-using AK::parse_ascii_base36_digit;
-using AK::parse_ascii_digit;
-using AK::parse_ascii_hex_digit;
-using AK::to_ascii_base36_digit;
-using AK::to_ascii_lowercase;
-using AK::to_ascii_uppercase;
+
+
+
```

Comment:
Replacing these using declarations with newlines is definitely a statement, but did you intend to delete the newlines after? 😬

Reply:
(extraneous newlines show up in other files too)

Reply:
I decided I'd be lazy and remove them later, later never came. will fix